docs/rich-text.adoc

```diff
@@ -0,0 +1,81 @@
+= Rendering a person
+:toc: right
+:imagesdir: media/
+:sourcedir: ../src/main/resources/
+:snippet: ../code-samples/code-snippets/
+
+In this chapter you'll learn how to render rich text content.
+
+== Intro
+
+Enonic boasts a https://developer.enonic.com/docs/xp/stable/cms/schemas/input-types/htmlarea[rich text input type].
+Unlike basic text fields, rich text may contain more complex concepts such as tables, images, links and custom components.
+Each of these will typically need to be processed during rendering in order to provide a contextually correct URL, ideal image sizes etc.
+
+== Sample content
+
+Luckily, the sample data set contains everyting we need to get going.
+The `Bio` field for persons is rich text, and some of the content is even populated with relevant content, like you see below:
+
+image::rich-text-bio.png[title="Rich text bio field for Lea Seydoux",width=662px]
+
+== Rich text rendering
+
+. *Update the Person processor*
++
+./src/main/resources/react4xp/components/content/PersonProcessor.ts
+[source,TypeScript]
+----
+include::{snippet}Rich-text/PersonProcessor.ts[]
+----
+
```

Comment:
Add description about `processHtml` function. `processHtml` is a function to prepare data for `RichText` component that comes with react4xp.